src/Codeception/Module/PhpBrowser.php

```diff
@@ -143,17 +154,17 @@ public function _getUrl()
      * @param $name
      * @param $value
      */
-    public function setHeader($name, $value)
```

Comment:
Parameters of updated methods are strings.
I guess they depend on interface change too.